WS2012R2/lisa/setupscripts/ProvisionSshKeys.ps1

```diff
@@ -170,7 +173,8 @@ function InstallPackagesRequiredByLisa([String] $ipv4, [String] $password)
     switch -regex ($distro)
     {
         CentOS   { $cmd = "yum -y install dos2unix at"                        }
-        RedHat   { $cmd = "yum -y install dos2unix at"                        }
+        RedHat6   { $cmd = "yum -y install dos2unix at"                        }
+        RedHat7   { $cmd = "yum -y install dos2unix at"                        }
```

Comment:
We should keep the general RedHat tag for all versions from 5.x to 7.x. This is also to keep the compatibility with the current scripts that use the RedHat distro regex.
A 3rd reason is to remove duplicate code like the same commands for RHEL 6.x and 7.x if nothing is changed.